src/services/ldap/hooks/index.js

```diff
@@ -1,33 +1,33 @@
-const { authenticate } = require('@feathersjs/authentication');
-const { disallow, iffElse, isProvider } = require('feathers-hooks-common');
-const { hasPermission, populateCurrentSchool } = require('../../../hooks');
-const restrictToSchoolSystems = require('./restrictToSchoolSystems');
-
-exports.before = {
-	all: [],
-	find: [disallow()],
-	get: [
-		iffElse(isProvider('external'), [
-			authenticate('jwt'),
-			hasPermission('SYSTEM_EDIT'),
-			populateCurrentSchool,
-			restrictToSchoolSystems,
-		], [
-			disallow(),
-		]),
-	],
-	create: [disallow()],
-	update: [disallow()],
-	patch: [disallow()],
-	remove: [disallow()],
-};
-
-exports.after = {
-	all: [],
-	find: [],
-	get: [],
-	create: [],
-	update: [],
-	patch: [],
-	remove: [],
-};
+const { authenticate } = require('@feathersjs/authentication');
+const { disallow, iffElse, isProvider } = require('feathers-hooks-common');
+const { hasPermission, populateCurrentSchool } = require('../../../hooks');
+const restrictToSchoolSystems = require('./restrictToSchoolSystems');

```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5713943)